srcpkgs/gtksourceview5/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'gtksourceview5'
 pkgname=gtksourceview5
-version=5.2.0
+version=5.4.0
 revision=1
 wrksrc="gtksourceview-${version}"
 build_style=meson
 build_helper="gir"
 configure_args="-Dintrospection=$(vopt_if gir enabled disabled)
  $(vopt_bool gir vapi) $(vopt_bool gtk_doc gtk_doc)"
 hostmakedepends="glib-devel itstool libxslt pkg-config gtk4 gettext
- $(vopt_if gir vala) $(vopt_if gtk_doc gtk-doc)"
+ $(vopt_if gir vala) $(vopt_if gtk_doc gi-docgen)"
```

Comment:
```suggestion
 $(vopt_if gir vala) $(vopt_if gi-docgen)"
```
and split this to a separate PR if possible

Reply:
I'm confused by your suggested change. btw, I already fixed all the `$(vopt_if gtk_doc 'gtk-doc gi-docgen')` if that's what you meant.
Aside from that, sure.

Reply:
My bad, sorry.